apps/projects/app/components/Panel/PanelComponents.js

```diff
@@ -40,14 +42,111 @@ export const IssueTitle = ({ issue }) => (
 )
 IssueTitle.propTypes = issueShape
 
-export const IssueText = styled.div`
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+export const Avatar = ({ user }) => {
+  const theme = useTheme()
+
+  return (
+    <Link
+      href={user.url}
+      target="_blank"
+      style={{ textDecoration: 'none', color: `${theme.link}`, marginRight: 6 }}
+    >
+      <img
+        alt=""
+        src={user.avatarUrl}
+        css="width: 40px; height: 40px; margin-right: 10px; border-radius: 50%;"
+      />
+    </Link>
+  )
+}
+Avatar.propTypes = {
+  user: PropTypes.shape({
+    url: PropTypes.string.isRequired,
+    avatarUrl: PropTypes.string.isRequired,
+  })
+}
+
+const statuses = [
+  { color: 'negative', text: 'Rejected' },
+  { color: 'positive', text: 'Accepted' },
+]
+
+export const Status = ({ review }) => {
+  const theme = useTheme()
+  const status = statuses[Number(review.approved)]
+  const text = status.text
+  const color = theme[status.color]
+  return (
+    <div css="display: flex; align-items: center">
+      <IconCheck color={color} css={`margin-top: -${0.5 * GU}px; margin-right: ${GU}px`} />
+      <Text color={`${color}`}>{text}</Text>
+      <Text color={`${theme.contentSecondary}`} css={`margin-left: ${GU}px`}>
+        {formatDate(new Date(review.reviewDate), 'd MMM yy, h:MM a (z)')}
+      </Text>
+    </div>
+  )
+}
+Status.propTypes = {
+  review: PropTypes.shape({
+    approved: PropTypes.bool.isRequired,
+    reviewDate: PropTypes.string.isRequired,
+  }),
```

Comment:
`isRequired`?

Reply:
Yes. Both are used, both are required...

Reply:
But is `review` itself required? If so, we can add `isRequired` at the end here, too.

```js
  review: PropTypes.shape({
    approved: PropTypes.bool.isRequired,
    reviewDate: PropTypes.string.isRequired,
  }).isRequired,
```

Reply:
Fixeth in both places.